repose-aggregator/components/filters/ip-user-filter/src/test/scala/org/openrepose/filters/ipuser/IpUserFilterTest.scala

```diff
@@ -159,6 +159,17 @@ class IpUserFilterTest extends FunSpec with BeforeAndAfterEach with Matchers {
     }
   }
 
+  describe("when presented with non-sense") {
+    it("should return a 400 when X-Forwarded-For is malformed") {
+      servletRequest.setRemoteAddr("10.1.2.3")
+      servletRequest.addHeader("X-Forwarded-For", "banana-phone")
+
+      ipUserFilter.doFilter(servletRequest, servletResponse, filterChain)
+
+      servletResponse.getStatus shouldBe 400
```

Comment:
The `400` could be replaced by `HttpServletResponse.SC_BAD_REQUEST`. However, I'm not sure I care since this is in a test. In fact, I wonder if using `400` was intentional to match the test name.